docs/node-operators/how-to/maintain-multisig-contract.md

```diff
@@ -0,0 +1,71 @@
+---
+sidebar_position: 10
+title: Maintaining the Multisig Contract
+sidebar_label: Maintaining Multisig Contract
+hide_title: false
```

Comment:
```suggestion
hide_title: false
vega_network: TESTNET
```

This is required for the `NetworkParameter` component I will be proposing using below.